src/cmds/scripts/pbs_postinstall.in

```diff
@@ -469,6 +469,11 @@ elif [ $INSTALL_PACKAGE != client ] ; then
 			[ -f /etc/profile.d/pbs.csh ] || cp ${PBS_EXEC}/etc/pbs.csh /etc/profile.d
 			[ -f /etc/profile.d/pbs.sh ] || cp ${PBS_EXEC}/etc/pbs.sh /etc/profile.d
 		fi
+		unitfilepath="/usr/lib/systemd/system/"
+		if [ -d $unitfilepath ]; then
+			cp ${PBS_EXEC}/libexec/pbs.service $unitfilepath
```

Comment:
Should this file be copied by the postinstall script or installed as part of the RPM? Note that pbs.csh and pbs.sh are installed via the RPM for the OSS release (refer to src/cmds/scripts/Makefile.am). The cp commands are still present in the postinstall script to accommodate the legacy installation method. The advantage of installing it via the RPM is that it will be deleted if the RPM is uninstalled. Otherwise, we should update pbspro.spec to remove this file as part of the %preun steps.